spec.bs

```diff
@@ -1340,6 +1343,49 @@ Note: This is because we need to ensure that we do not leak <var ignore>creator<
 document's referrer|referrer=], [=Document/origin=], [=creator base url=], [=Document/policy
 container=], across the fenced frame boundary.
 
+<h3 id=local-policy-container-monkeypatch>Local Requests Policy Container Patch</h3>
+
+When making local requests, <{iframe}>s inherit their [=Document/policy container=] from their
+[=navigable container=]'s [=Node/node document=]. If <{fencedframe}>s were to do the same thing,
+that would allow information about the initiator's [=IP address space=] to leak across a fenced
+frame boundary. This section patches [=Document/policy container=] inheritance to close that leak.
+
+<div algorithm=local-policy-inheritance-determine-params>
+  Modify the definition of the [=determine navigation params policy container=] algorithm to read:
+
+  To determine navigation params policy container given a [=URL=] |responseURL|; four [=policy
+  container=]-or-nulls <var ignore>historyPolicyContainer</var>, |initiatorPolicyContainer|,
+  <var ignore>parentPolicyContainer</var>, and <var ignore>responsePolicyContainer</var>; and an
+  optional [=boolean=] |fenced| that defaults to false:
+
+  Rewrite step 3 to read:
+
+    3. If |responseURL| is local, |initiatorPolicyContainer| is not null, and |fenced| is set to
+       false, then return a [=clone a policy container|clone=] of |initiatorPolicyContainer|.
```

Comment:
Add a note below this algorithm to mention that we do not have to modify the `about:srcdoc` case (and please link to the definition correctly with the right formatting: https://html.spec.whatwg.org/#about:srcdoc) because fenced frames cannot navigate to srcdoc resources.